assets/src/components/PopupMessage.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react'
 import { Typography } from '@material-ui/core'
+import { roundToXDecimals } from '../util/math'
 
 function PopupMessage ({ a, assignmentGroups }) {
   const getAssignmentRules = (a, assignmentGroups) => {
```

Comment:
@ssciolla On line#7 there is a typo in the const ` const assignmenGroup = assignmentGroups.find(ag => ag.id === a.assignmentGroupId)`. Can you fix that as well since your PR is modifying that file? and it is also a small change to fix

Reply:
Should be fixed now, @pushyamig .